src/components/selection-list/selection-list.st.css

```diff
@@ -1,13 +1,15 @@
 @namespace "SelectionList";
 
-:import {
-    -st-from: "../../style/default-theme/variables.st.css";
-    -st-named: borderFocus;
-}
-
 .list {
+    -st-states: focused;
     width: 276px;
-    border: value(borderFocus);
+    border: 1px solid #adbbc6;
+    outline: none;
+}
+
+.divider {
+    height: 1px;
+    background-color: #adbbc6;
```

Comment:
@idoros Right, there are probably no styles that we should apply to the component by default. I started refactoring styling, and then got stuck with trying to fix other related problems without deviating from the spec too much.